backend/models/users.js

```diff
@@ -12,6 +12,15 @@ const users = new Schema({
   favoriteIds: {
     type: Array,
   },
+  username: {
+    type: String,
+  },
+  email: {
+    type: String,
+  },
+  password: {
+    type: String,
+  },
```

Comment:
Consider making username and password required fields in the Mongo schema with the required: true field